go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/containers/image/v5 v5.4.4
 	github.com/containers/libpod v1.9.2
 	github.com/containers/ocicrypt v1.0.2
+	github.com/containers/psgo v1.4.0
```

Comment:
Hmm, there should also be an entry in `go.sum` which I don't see in this commit, as well as some new files under `vendor/`.